src/ObjectMakr.ts

```diff
@@ -226,15 +225,17 @@ export class ObjectMakr implements IObjectMakr {
         // For each name in the current object:
         for (let name in base) {
             if (base.hasOwnProperty(name)) {
```

Comment:
While you're here, can you change this to continue if `@base.hasOwnProperty(name)` so the rest of the stuff in the for loop has one less level of indentation?

I wrote this code a very long time ago. :)